docs/designers-developers/developers/tutorials/create-block/submitting-to-block-directory.md

```diff
@@ -0,0 +1,102 @@
+# Share your Block with the World
+
+So you've created an awesome block? Care to share?
+
+**Contents**:
+1. Help users understand your block
+2. Analyze your plugin
+3. Zip & Submit
+
+## Step 1: Help users understand your block
+
+Providing straightforward, easy to understand block information is important to the block directory and our end users.
+
```

Comment:
It is important to the Block Directory and our end users to provide easy to understand information on how your block was created.